client/src/components/LandingPage.jsx

```diff
@@ -18,32 +19,41 @@ function LandingPage() {
   const [Lunchlist, setLunchList] = useState([]);
   const [Dinnerlist, setDinnerList] = useState([]);
   const [data,setData] = useState([])
+  const { loginWithRedirect, isAuthenticated, logout , user } = useAuth0();
+  // const { logout } = useAuth0();
+  // const { , isLoading } = useAuth0();
+  console.log(user)
 
   useEffect(() => {
     // console.log(Breakfastlist);
     // console.log(Lunchlist);
     // console.log(Dinnerlist);
     const getCourse = async () => {
       try {
-        // const res = await fetch('https://balancebox.onrender.com/get-food');
-        const res = await fetch('http://localhost:5000/get-food')
+        const res = await fetch('https://balancebox.onrender.com/get-food');
```

Comment:
Add `https://balancebox.onrender.com` to a variable and use it, keep variable in `.env`